### PR TITLE
test: Edge cases - Smart Contracts Service Tests Part 2

### LINKED ISSUE #19760: Edge cases - Smart Contracts Service Tests

### PR DESCRIPTION
**Pull request by vtronkov**
**Description**:
* Add EVM tests wrapped inside an atomic batch

**Related issue(s)**:

Fixes #19760

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/evm/batch/AtomicEvm46ValidationSuite.java [hunk 1]
```diff
@@ -0,0 +1,1515 @@
...
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.stream.Stream;
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Tag;
+
+// This test cases are direct copies of Evm38ValidationSuite. The difference here is that
```
Comment (mhess-swl) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/evm/batch/AtomicEvm46ValidationSuite.java:83]:
```suggestion
// This test cases are direct copies of Evm46ValidationSuite. The difference here is that
```

Also missed in `AtomicEvm50ValidationSuite`. Can be done in a future PR

**Review by mhess-swl — APPROVED** [review]
